src/mono/wasm/runtime/package.json

```diff
@@ -3,7 +3,7 @@
   "description": ".NET is a developer platform with tools and libraries for building any type of app, including web, mobile, desktop, games, IoT, cloud, and microservices.",
   "repository": {
     "type": "git",
-    "url": "git@github.com:dotnet/runtime.git"
+    "url": "https://github.com/dotnet/runtime"
```

Comment:
```suggestion
    "url": "https://github.com/dotnet/runtime.git"
```

The `.git` suffix in the URL to clone is still being used.

![image](https://user-images.githubusercontent.com/12659251/200184549-b1b027d7-37b6-415c-aa8a-8a2290b41b76.png)

Reply:
@teo-tsirpanis Do you know if there is any difference to using URL without `.git` suffix? Both of them works...

Reply:
Don't know, guess it's just a convention.

Reply:
Ok, thanks